### ISSUE TITLE
Remove Zero Weight bones: list option and/or only operate on bones with no children option.

### ISSUE THREAD

**Issue by hanetzer** [teamneoneko/Cats-Blender-Plugin-Unofficial-#172]
As title. Basic gist, if you use this option as is, you may nuke your parent bones for things like skirts, ears, or breasts.

It would be nice to have a tickbox to prevent deletion of bones with children (if you're using it to remove extra leaf
bones, or remove bones from clothing items or similar mesh removals) or just make a list of bones with zero weight
for manual removal. Also a good idea for a feature for avatar toolkit.

**Comment by Yusarina** [comment]
Hey, 

Sorry for the late reply on this one, this feature won't be added to Cats. We have decided unless the feature is a small add, no more features will be Added to cats. This is because we are working on Avatar Toolkit. 

However this feature will be considered for Avatar Toolkit instead, I have added the tag for this and once the Core toolkit is done will transfer this issue over to it.

Thanks!

**Comment by hanetzer** [comment]
Sounds good. And, maybe another thing. have a toggle to either target only deform bones, or non-deform bones.
And, similarly, mark parent's made with cat's util as being non-deform.

**Comment by Yusarina** [comment]
We will not add this to cats, however I posted it on Avatar Toolkit now, see https://github.com/teamneoneko/Avatar-Toolkit/issues/61

Due to this I have closed this issue.
